src/components/Navigation/README.md

```diff
@@ -202,7 +202,7 @@ Use to present a secondary action, related to a section and to title the section
   <Navigation.Section
     items={[
       {
-        url: '/',
+        url: '/path/to/place',
```

Comment:
We had two active items in this example which should never happen. Undoing this one since the example if for selected secondary, which is below